Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -0,0 +1,211 @@
+
+    /// <summary>
+    // Main idea: Simulate genetic mutation using random binary flips.  Each
+    // seed attribute can be encoded with a variable number of bits, e.g.
+    // NutrientConsumption is represented by 5 bits randomly distributed in the
+    // plant's genome which thermometer code the floating value between 0.1 and
+    // 5. 1 unit of mutation flips one bit in the plant's genome, which changes
+    // NutrientConsumption if one of those 5 bits gets affected.
+    //
+    // You MUST clone() seed before mutating it!
+    /// </summary>
+    public static void MutateSeed(SeedData seed, float severity)
+    {
+        // Add up everything in the bits column and put the number here.
+        const int totalbits = 215;
+
+        // Tolerances (55)
+        MutateFloat(ref seed.NutrientConsumption   , 0.05f , 1.2f , 5 , totalbits , severity);
+        MutateFloat(ref seed.WaterConsumption      , 3f    , 9f   , 5 , totalbits , severity);
+        MutateFloat(ref seed.IdealHeat             , 263f  , 323f , 5 , totalbits , severity);
+        MutateFloat(ref seed.HeatTolerance         , 2f    , 25f  , 5 , totalbits , severity);
+        MutateFloat(ref seed.IdealLight            , 0f    , 14f  , 5 , totalbits , severity);
+        MutateFloat(ref seed.LightTolerance        , 1f    , 5f   , 5 , totalbits , severity);
+        MutateFloat(ref seed.ToxinsTolerance       , 1f    , 10f  , 5 , totalbits , severity);
+        MutateFloat(ref seed.LowPressureTolerance  , 60f   , 100f , 5 , totalbits , severity);
+        MutateFloat(ref seed.HighPressureTolerance , 100f  , 140f , 5 , totalbits , severity);
+        MutateFloat(ref seed.PestTolerance         , 0f    , 15f  , 5 , totalbits , severity);
+        MutateFloat(ref seed.WeedTolerance         , 0f    , 15f  , 5 , totalbits , severity);
+
+        // Stats (30*2 = 60)
+        MutateFloat(ref seed.Endurance             , 50f   , 150f , 5 , totalbits , 2*severity);
+        MutateInt(ref seed.Yield                   , 3     , 10   , 5 , totalbits , 2*severity);
+        MutateFloat(ref seed.Lifespan              , 10f   , 80f  , 5 , totalbits , 2*severity);
+        MutateFloat(ref seed.Maturation            , 3f    , 8f   , 5 , totalbits , 2*severity);
+        MutateFloat(ref seed.Production            , 1f    , 10f  , 5 , totalbits , 2*severity);
+        MutateFloat(ref seed.Potency               , 30f   , 100f , 5 , totalbits , 2*severity);
+
+        // Kill the plant (30)
+        MutateBool(ref seed.Viable         , false , 30 , totalbits , severity);
+
+        // Fun (70)
+        MutateBool(ref seed.Seedless       , true  , 10 , totalbits , severity);
+        MutateBool(ref seed.Slip           , true  , 10 , totalbits , severity);
+        MutateBool(ref seed.Sentient       , true  , 10 , totalbits , severity);
+        MutateBool(ref seed.Ligneous       , true  , 10 , totalbits , severity);
+        MutateBool(ref seed.Mysterious     , true  , 10 , totalbits , severity);
```

Comment:
i dont think mysterious is meant to be altered

Reply:
I'll remove mysterious alteration.